idaes/gas_solid_contactors/unit_models/bubbling_fluidized_bed.py

```diff
@@ -1073,11 +1191,20 @@ def bubble_cloud_heat_trans_coeff(b, t, x):
                              self.length_domain,
                              doc="Convective Heat Transfer Coefficient")
             def convective_heat_trans_coeff(b, t, x):
+                # 0.03 (kg/mol)**1.3 is a unitted constant in this correlation
+                # unit containers moved to the LHS of equation for stability
+                reform_var_5_units = (units_meta_gas('amount') /
+                                      units_meta_gas('mass'))
                 return (
-                    1e6*b.htc_conv[t, x] *
-                    b.solid_emulsion.properties[t, x]._params.particle_dia ==
-                    1e6 * 0.03 * b.gas_emulsion.properties[t, x].therm_cond *
-                    ((b._reform_var_5[t, x]**2 + b.eps)**0.5) ** 1.3)
+                    b.htc_conv[t, x] *
+                    ((pyunits.mol/pyunits.kg)**1.3) *
+                    pyunits.convert(b.solid_emulsion.properties[t, x]
+                                    ._params.particle_dia,
+                                    to_units=units_meta_gas('length')) ==
+                    0.03 *
+                    b.gas_emulsion.properties[t, x].therm_cond *
+                    ((b._reform_var_5[t, x]**2 + b.eps*reform_var_5_units**2
```

Comment:
1. Why does `eps` appear here?
2. Note that once `eps` gets units added you will need different `eps` objects for each place it is used.